luma/oled/device/framebuffer_mixin.py

```diff
@@ -0,0 +1,24 @@
+import luma.core.framebuffer
```

Comment:
copyright header missing

Reply:
and version marker in the docstring (of the module).

Reply:
fixed